src/lib.rs

```diff
@@ -1908,6 +1897,13 @@ pub(crate) mod test {
                 Complex::new(1.65826, -0.33502),
                 1e-5
             ));
+            let z = Complex::new(0.0, 0.0);
+            assert!(close(z.powc(b), z));
+            assert!(z.powc(Complex64::new(0., 0.)).is_nan());
+            assert!(z.powc(Complex64::new(0., INFINITY)).is_nan());
+            assert!(z.powc(Complex64::new(10., INFINITY)).is_nan());
+            assert!(z.powc(Complex64::new(INFINITY, INFINITY)).is_nan());
+            assert!(close(z.powc(Complex64::new(INFINITY, 0.)), z));
```

Comment:
```suggestion
            assert!(close(z.powc(Complex64::new(INFINITY, 0.)), z));
            assert!(z.powc(Complex64::new(-1., 0.)).is_infinite());
```

Reply:
This one is a tricky case as it is actually `infinity + iNaN`, which returns false for `is_infinite()`. But I think the return value is fine though, so I would suggest checking
```
 assert!(z.powc(Complex64::new(-1., 0.)).re.is_infinite());
 assert!(z.powc(Complex64::new(-1., 0.)).im.is_nan());
```

Reply:
@konsumlamm Would you be happy with these altered tests? Or do you think in this case the function should return something like `infinity + i infinity`?

Reply:
Yes, I'm happy, feel free to resolve  this.